app/lib/controlled_work_mappings/cw5_header.yml

```diff
@@ -21,325 +21,389 @@
 
 # Savings / Client - "Savings (bank, building society, etc)" (non-SMOD)
 - name: FillText110
   type: text
+  section: capital
   source: non_smod_client_savings
 
 # Savings / Partner - "Savings (bank, building society, etc)" (non-SMOD)
 - name: FillText114
   type: text
+  section: partner_capital
   source: partner_savings
 
 # Investments / Client - "Investments (shares, insurance policies, etc)" (non-SMOD)
 - name: FillText111
   type: text
+  section: capital
   source: non_smod_client_investments
 
 # Investments / Partner - "Investments (shares, insurance policies, etc)" (non-SMOD)
 - name: FillText115
   type: text
+  section: partner_capital
   source: partner_investments
 
 # Valuables / Client - "Valuable items (boat, caravan, jewellery, etc)" (non-SMOD)
 - name: FillText112
   type: text
+  section: capital
   source: non_smod_client_valuables
 
 # Valuables / Partner - "Valuable items (boat, caravan, jewellery, etc)" (non-SMOD)
 - name: FillText116
   type: text
+  section: partner_capital
   source: partner_valuables
 
 # Total capital part B (non-SMOD)
 - name: FillText118
   type: text
+  section: capital
   source: combined_non_disputed_capital
 
 # Total of all capital assets (Part A and Part B)
 - name: FillText119
   type: text
+  section: capital
   source: combined_assessed_capital
 
 # Part C
 # The client is directly or indirectly in receipt of [passporting benefit]
 - name: Passported
   type: boolean_radio
+  section: income
   source: passporting
   yes_value: "Yes"
   no_value: "No"
 
 # Gross monthly earnings / Client
 - name: FillText82
   type: text
+  section: income
   source: client_gross_income
 
 # Gross monthly earnings / Partner
 - name: FillText83
   type: text
+  section: partner_income
   source: partner_gross_income
 
 # Other income / Client
 - name: FillText84
   type: text
+  section: income
   source: client_non_employment_income
 
 # Other income / Partner
 - name: FillText85
   type: text
+  section: partner_income
   source: partner_non_employment_income
 
 # Total gross income / Client
 - name: FillText86
   type: text
+  section: income
   source: client_total_income
 
 # Total gross income / Partner
 - name: FillText94
   type: text
+  section: partner_income
   source: partner_total_income
 
 # Total gross income  Client and Partner
 - name: FillText98
   type: text
+  section: income
   source: total_combined_income
 
 # Mortgage instalment / Client
 - name: FillText35
   type: text
+  section: income
   source: client_mortgage
 
 # Mortgage instalment / Partner
 - name: FillText36
   type: text
+  section: partner_income
   source: partner_mortgage
 
 # Rent / Client
 - name: FillText37
   type: text
+  section: income
   source: client_rent
 
 # Rent / Partner
 - name: FillText38
   type: text
+  section: partner_income
   source: partner_rent
 
 # Dependents' allowances / Partner
 - name: FillText39
   type: text
+  section: income
```

Comment:
should this be `partner_income`?

Reply:
@HettieS I don't think it should. The partner allowance is not in the partner section, it's in the client section. If the client doesn't have a partner I don't think it should be left blank, I think it should be zero (in the same way that if the client doesn't have child dependants the child dependant field is left zero).

Conceptually, it's because  if the client doesn't have a partner, "How much partner allowance does the client get?" is still a meaningful question, while, for example, "What are the partner's gross monthly earnings?" is not.

Reply:
Ooooh I see. My mistake for not looking at the forms properly. I noticed for partner dependant allowance, in some forms you have `partner_income` and others you have `income` - so on cw5 it's `income` and in cw2 it is `partner_income` for example. I'm just looking at the forms trying to work it out....

Reply:
Ohhh, my bad, so yeah, I got it wrong on CW2. Fixing now.